server/src/main/java/org/candlepin/model/Pool.java

```diff
@@ -1262,8 +1273,8 @@ public PoolType getType() {
             if (hasAttribute(Attributes.UNMAPPED_GUESTS_ONLY)) {
                 return PoolType.UNMAPPED_GUEST;
             }
-            else if (hasAttribute(Attributes.SHARE)) {
-                return PoolType.SHARE_DERIVED;
+            else if (hasAttribute(Attributes.SHARED_POOL)) {
```

Comment:
hasAttribute and value = 'true'?
I realize you just substituted the tag, but the hasAttribute test seems insufficient. Same for unmapped guest condition.